src/lib/components/data-table/DataTableActions.svelte

```
@@ -65,69 +82,146 @@
+						class="bg-[hsla(242,67%,90%,1)] text-indigo-700 hover:bg-[hsla(242,67%,85%,1)]"
+						onclick={action.action}
+					>
 						{#if IconComponent}
-							<IconComponent />
+							<IconComponent class="h-4 w-4" />
 						{/if}
 						{action.label}
-					</DropdownMenu.Item>
+					</Button>
 				{/if}
 			{/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)